cmd/etcd.go

```diff
@@ -61,134 +61,171 @@ func newShowDDLInfoCommand() *cobra.Command {
 	return m
 }
 
-// newDelOwnerCampaign return a delete owner campaign subcommand of EtcdCommand
-func newDelOwnerCampaign() *cobra.Command {
+// newDelKeyCommand returns a delete key subcommand of EtcdCommand.
+func newDelKeyCommand() *cobra.Command {
 	m := &cobra.Command{
-		Use:   "delowner [LeaseID]",
-		Short: "delete DDL Owner Campaign by LeaseID",
-		Run:   delOwnerCampaign,
+		Use:   "delkey",
+		Short: "del key by `delkey [key]`",
+		Run:   delKeyCommandFunc,
 	}
 	return m
 }
 
-// newDelSchemaVersion return a delete schema version subcommand of EtcdCommand
-func newDelSchemaVersion() *cobra.Command {
+// newPutKeyCommand returns a put key subcommand of EtcdCommand.
+func newPutKeyCommand() *cobra.Command {
 	m := &cobra.Command{
-		Use:   "delschema",
-		Short: "delete schema version by DDLID",
-		Run:   delSchemaVersion,
+		Use:   "putkey",
+		Short: "[ONLY FOR TEST!] put key in schema version by `putkey [key] [value]`",
+		Run:   putKeyCommandFunc,
 	}
 	return m
 }
 
 func showDDLInfoCommandFunc(cmd *cobra.Command, args []string) {
-	var rangeQueryDDLInfo = &parameter{
-		Key:      base64Encode("/tidb/ddl"),
-		RangeEnd: base64Encode("/tidb/ddm"),
-	}
-
-	reqData, err := json.Marshal(rangeQueryDDLInfo)
+	res, err := getDDLInfo()
 	if err != nil {
 		cmd.Printf("Failed to show DDLInfo: %v\n", err)
 		return
 	}
-	req, err := getRequest(cmd, rangeQueryPrefix, http.MethodPost, "application/json",
-		bytes.NewBuffer(reqData))
-	if err != nil {
-		cmd.Printf("Failed to show DDLInfo: %v\n", err)
+	cmd.Println(res)
+}
+
+func delKeyCommandFunc(cmd *cobra.Command, args []string) {
+	if len(args) != 1 {
+		cmd.Printf("Only one argument!")
 		return
 	}
-	res, err := dail(req)
-	if err != nil {
-		cmd.Printf("Failed to show DDLInfo: %v\n", err)
+
+	key := args[0]
+	ddlOwnerKeyPrefix := "/tidb/ddl/fg/owner/"
+	ddlAllSchemaVersionsPrefix := "/tidb/ddl/all_schema_versions/"
+	if !(strings.HasPrefix(key, ddlOwnerKeyPrefix) || strings.HasPrefix(key, ddlAllSchemaVersionsPrefix)) {
+		cmd.Printf("This function only for delete something about DDL")
```

Comment:
Do we need add `\n` at the end of this log? We can print the `args` information.

Reply:
I think print `args` is not necessary.
Because this error is usually close to what you input.